docs-site/content/overview/demos.md

```diff
@@ -16,3 +16,4 @@ Here are some live demos, that show Typesense used for different use cases.
 | [Federated Search](https://federated-search.typesense.org/)                                                              | 10K usernames and 10K company names | A site that showcases Typesense's Federated Search feature.                                                                                                               |
 | [AirBnB Geo Search](https://airbnb-geosearch.typesense.org/)                                                             | 1.2M AirBnB listings                | A site that showcases Typesense's Geo Search feature.                                                                                                                     |
 | [Semantic Search + Hybrid Search](https://hn-comments-search.typesense.org/)                                             | 300K Comments from HackerNews       | A site that showcases Typesense's semantic search and hybrid search features, along with the auto-embedding generation feature.                                           |
+| Guitar Chords Search                                                                                                     | 2141 chord shapes of 552 chords     | Sites that showcase Typesense in multiple Javascript frameworks: [NuxtJS](https://showcase-guitar-chords-search-nuxt-js.typesense.org/), [NextJS](https://showcase-guitar-chords-search-next-js.typesense.org/), [Angular](https://showcase-guitar-chords-search-angular.typesense.org/), and [Vanilla JS](https://showcase-guitar-chords-search-vanilla-js.typesense.org/) variations.  |
```

Comment:
Guitar Chords Search _in_ multiple Javascript framworks. (Could you add the "in")